dev/preview/previewctl/pkg/k8s/vm.go

```diff
@@ -163,3 +162,29 @@ func (c *Config) GetVMs(ctx context.Context) ([]string, error) {
 
 	return vms, nil
 }
+
+func (c *Config) getVMI(ctx context.Context, name, namespace string) (*virtv1.VirtualMachineInstance, error) {
+	_, err := c.coreClient.CoreV1().Namespaces().Get(ctx, namespace, metav1.GetOptions{})
+	if err != nil && !kerrors.IsNotFound(err) {
+		return nil, errors.Wrap(ErrVmNotReady, err.Error())
+	}
+
+	if kerrors.IsNotFound(err) {
+		c.logger.Infof("namespace [%s] not found", namespace)
+		return nil, errors.Wrap(ErrVmNotReady, err.Error())
+	}
```

Comment:
I failed to understand what the first if is doing 🤔 

Your conditionals are contradictory but returning the same errors? 🤔

Reply:
This is cut-pasted as part of the refactoring to extract shared code into a getVMI function so I didn't look at it too closely, but the only difference I see is the additional logging being done ☺️ I'm inclined to keep it as is and we can return to it when Aleks is back from vacation.